apps/remix-ide/src/app/tabs/runTab/settings.js

```diff
@@ -53,6 +53,17 @@ class SettingsUI {
     })
   }
 
+  validateValue () {
+    this.el.querySelector('#value').value = parseInt(this.el.querySelector('#value').value)
```

Comment:
Why don't you create a variable for `this.el.querySelector('#value').value` and use the variable instead for your logic.

Reply:
I'm changing the value in each step